editoast/src/client/mod.rs

```diff
@@ -67,6 +69,11 @@ pub enum SearchCommands {
     Refresh(RefreshArgs),
 }
 
+#[derive(Subcommand, Debug)]
+pub enum InfraCommands {
+    Clone(InfraCloneArgs),
```

Comment:
This is a good idea to create a subcommand for infra manipulation. If you do so please move the following commands:

- Generate
- Clear
- ImportRailjson

**Note:** you must adapt scripts that are using editoast command line. Checkout [scripts](https://github.com/osrd-project/osrd/tree/dev/scripts)

Reply:
If you don't want to make this modification in this PR, can you remove the subcommand?

Flatten this command.

Reply:
If we remove the subcommand, the command will be less explicit, like `editoast infra <id>`, compared to the more explicit `editoast infra clone <id>`

Reply:
Agreed with @flomonster. You're right, if you decide to remove the subcommand, you need to rename it "infra-clone" or something similar. But since you've already setup the subcommand, you might as well move the commands Florian listed above under `editoast infra <cmd>`.

Reply:
I've just moved the `clear` command to the subcommands in this PR. I'll move the others in a separate PR later. Is that okay for you?